config

```diff
@@ -1,5 +1,5 @@
 {
-	"AccessToken":["YourBitlyAccessTokenHere"],
+	"AccessToken":["Yf7d318a38ee40a5049861a32814bb1b0fb78cb8b"],
```

Comment:
Do we want to provide an access token in the source? This token will perhaps be revoked by GitHub. It is better to tell user how to create a token and then they can use that instead.